samples/shields/x_nucleo_iks01a3/standard/app.overlay

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2024 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * Sensors declared here are possibly present on DIL24
+ */
+&arduino_i2c {
+
```

Comment:
drop? :-)

Reply:
My usual mistakes, right Fabio? :D

Reply:
ah this is code review stuff, not a problem :-)